libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html

```diff
@@ -28,6 +28,50 @@
     label="{{
       'dmp.steps.project.input.description' | translate
     }}"></app-textarea-wrapper>
+
+  <div formGroupName="funding">
+    <div class="form-row" formGroupName="funderId">
+      <app-input-wrapper-options
+        class="input-60"
+        [control]="funder"
+        [options]="funderOptions"
+        label="{{ 'Funder' | translate }}"></app-input-wrapper-options>
+
+      <mat-form-field appearance="fill">
```

Comment:
Non-textarea form fields should have appearance="outline" to keep in sync with the rest of the project.